ab2d-events-client/src/main/java/gov/cms/ab2d/eventclient/events/LoggableEvent.java

```diff
@@ -121,7 +121,7 @@ public boolean equals(final Object o) {
             if (otherTimeOfEvent != null) {
                 return false;
             }
-        } else if (thisTimeOfEvent.toEpochSecond() != otherTimeOfEvent.toEpochSecond()) {
+        } else if (otherTimeOfEvent == null || thisTimeOfEvent.toEpochSecond() != otherTimeOfEvent.toEpochSecond()) {
```

Comment:
I found this missing null check when writing the tests

Reply:
Good catch

Reply:
Do you plan to increase library version? may be later?

Reply:
Oh, thanks for catching that @smirnovaae! I forgot that part. I'll do it in another PR.